generated-sample-code/samplegen_no_response.js

```diff
@@ -0,0 +1,47 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// DO NOT EDIT! This is a generated sample ("Request",  "samplegen_no_response")
+
+// sample-metadata:
+//   title: Delete Product Set (returns Empty)
+//   description: Delete Product Set (returns Empty)
+//   usage: node samples/v1/samplegen_no_response.js
+
+'use strict';
+
+function main() {
+  // [START samplegen_no_response]
+
+  // Imports the client library
+  const {ProductSearchClient} = require('@google-cloud/vision').v1;
+
+  // Instantiates a client
+  const productSearchClient = new ProductSearchClient();
+
+  async function sampleDeleteProductSet() {
+
+    // The full name of the product set to delete
+    const formattedName = productSearchClient.productSetPath('[PROJECT]', '[LOCATION]', '[PRODUCT_SET]');
+
+    // Run request
+    const [response] = await productSearchClient.deleteProductSet({name: formattedName});
```

Comment:
If no response is expected, can we just do `await productSearchClient.deleteProductSet({name: formattedName});` without any assignment?

Reply:
Yassssssss! We can! w00t, noted!